packages/material-ui/src/Input/Textarea.d.ts

```diff
@@ -11,7 +11,7 @@ export interface TextareaProps
   disabled?: boolean;
   rows?: string | number;
   rowsMax?: string | number;
-  textareaRef?: React.Ref<any>;
+  textareaRef?: React.Ref<any> | React.RefObject<any>;
```

Comment:
This is not needed. `React.Ref<>` includes `React.RefObject<>`

> 
```ts 
type Ref<T> = string | { bivarianceHack(instance: T | null): any }["bivarianceHack"] | RefObject<T>;
```

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2f1c881b33b5944fd9383c33b792b37a2403a222/types/react/index.d.ts#L56

Reply:
`React.Ref<any> | React.RefObject<any>` can also be found in couple of other places, do you want to work on it @franklixuefei ? Otherwise I can make a PR to replace them.

Reply:
@t49tran Sorry I was busy working on something else lately. Please go ahead! Thanks! 😄